samples/cellular/modem_shell/README.rst

```diff
@@ -1366,16 +1497,26 @@ When connected, the ``location`` and ``gnss`` commands use the LwM2M cloud conne
 
 .. _modem_shell_trace_support:
 
-nRF9160 DK and modem trace support
-==================================
+nRF91 Series DK and modem trace support
+=======================================
 
-To build the MoSh sample with nRF9160 DK and modem traces with flash backend, use the ``-DDTC_OVERLAY_FILE=nrf9160dk_ext_flash.overlay`` and  ``-DOVERLAY_CONFIG="overlay-modem-trace-flash.conf;overlay-memfault.conf"`` options.
+To build the MoSh sample with an nRF91 Series DK and modem traces with flash backend, use ``-DDTC_OVERLAY_FILE=nrf9160dk_ext_flash.overlay`` for the nRF9160 DK and ``-DDTC_OVERLAY_FILE=nrf9161dk_ext_flash.overlay`` for the nRF9161 DK, and  ``-DOVERLAY_CONFIG="overlay-modem-trace-flash.conf;overlay-memfault.conf"`` options.
```

Comment:
```suggestion
To build the MoSh sample with an nRF91 Series DK and modem traces with flash backend, use the devicetree overlay for external flash corresponding to your device and the ``-DOVERLAY_CONFIG="overlay-modem-trace-flash.conf;overlay-memfault.conf"`` option.
```
Not sure that's an improvement, but the sentence is really long and unwieldy when all the three options are present. Since the full options are in the examples below, I don't think they're necessary here.